### ISSUE TITLE
Render private roads differently from public roads

### ISSUE THREAD

**Issue by ZeLonewolf** [ZeLonewolf/openstreetmap-americana#679]
Blocks #308

Currently, #308 is blocked by a lack of differentiated rendering of private roads. I was not able to find many examples online of paper maps that indicate private roads.

![image](https://user-images.githubusercontent.com/3254090/211881335-65d1a60d-d5e3-4719-b7f1-836245156073.png)

Google indicates private roads with a gray fill:
![image](https://user-images.githubusercontent.com/3254090/211881628-2744adcc-cdd7-459e-86d5-b05a934545d3.png)

osm-carto indicates private roads with a dash pattern in the fill:
![image](https://user-images.githubusercontent.com/3254090/211882164-b265002d-f62c-4400-a11b-7ebb0dceba75.png)

Are there other possible cartographic indications of no-access that we might consider?

Also, if we desire to render private roads at zooms where roads are represented as lines, we would need a separate solution for that if we go with any kind of fill pattern.

**Comment by 1ec5** [comment]
> Are there other possible cartographic indications of no-access that we might consider?

Most of the maps I have that show private or restricted-access roads simply append a gloss to the road name label (#503), such as “(Private)”. I think this is what lay users have come to expect, or else we wouldn’t see mappers so often hard-coding that parenthetical in road names in OSM. For situations where there isn’t enough space for both the road name and “(Private)”, we could have a separate symbol layer that labels just “(Private)” in fine print.

**Comment by ZeLonewolf** [comment]
> Most of the maps I have that show private or restricted-access roads simply append a gloss to the road name label (#503), such as “(Private)”

That's definitely a valid option.  Concerns we'd need to address:
* Localization (translation of the word private)
* What to show if a road doesn't have a name

**Comment by adamfranco** [comment]
If roads don't have name, do they still have an entry in the `transportation_name` layer? If so, a gloss-only like 
` (Private)` 
seems like it would be OK.

**Comment by adamfranco** [comment]
Unfortunately, using any sort of red fill would conflict with our `highway=trunk|motorway` coloring. The gray fill GM uses would conflict with our high-zoom unpaved rendering: 
![Screen Shot 2023-01-11 at 1 49 54 PM](https://user-images.githubusercontent.com/25242/211892680-ed7e0d04-3f35-4b04-9f17-4de09db8090d.png)

If workable, the gloss has the nice behavior of not conflicting with other attributes.

**Comment by ZeLonewolf** [comment]
> If roads don't have name, do they still have an entry in the `transportation_name` layer? If so, a gloss-only label like ` (Private)` seems like it would be OK.

No, unnamed roads are excluded from `transportation_name`, see:
https://github.com/openmaptiles/openmaptiles/blob/master/layers/transportation_name/update_transportation_name.sql#L42

**Comment by ZeLonewolf** [comment]
The question now is whether we're willing to have unnamed private roads be undifferentiated in terms of it's casing/fill style.

**Comment by adamfranco** [comment]
Hmm... This is also going to be an issue for not-yet-rendered private tracks and paths (#216). A solution for private roads should be at least a bit consistent with private tracks & paths, many [most?] of which won't be named.

**Comment by adamfranco** [comment]
Another possible option for private: Add a pink background underlay a pixel or so wider than the line that sets the road/path/track off from the background color slightly. 

Sort of like this "Closed in Winter" example, but narrower to make these less visually prominent.
![Screen Shot 2023-01-11 at 3 50 27 PM](https://user-images.githubusercontent.com/25242/211914923-d37d791f-12de-438a-b371-98d6372f0ac3.png)

Such an underlay/highlight wouldn't conflict with using a variety of dash patterns for paths and tracks to capture other properties (eg visibility/smoothness/informal/track-type).

**Comment by 1ec5** [comment]
If we go with a label, then the standalone “(Private)” symbol layer would be based on the `transportation` source layer`, not `transportation_name`. We’d have the normal transportation name symbol layer collide it out.
